resources/xml/views/dynamic_image.xml

```diff
@@ -0,0 +1,20 @@
+<brls:Box
+    width="auto"
+    height="auto"
+    axis="column"
+    alignItems="center">
+
+    <brls:Image
+        id="image"
+        width="auto"
+        height="auto"
+        focusable="true"
+        marginBottom="20px" />
+
+    <brls:Label
+        width="auto"
+        height="auto"
+        horizontalAlign="center"
+        text="Click the image!" />
```

Comment:
Please use i18n, and I think it would be better to have "Randomly generated RGB (click to refresh)" as a caption